product_docs/docs/pgd/5/quickstart/quick_start_docker.mdx

```diff
@@ -196,7 +196,7 @@ Linux as the default image for all nodes.
 
 Specify that the data nodes will be running [EDB Postgres Advanced Server v15](/epas/latest/) (`--edb-postgres-advanced 15`) with Oracle compatibility (`--redwood`). 
 
-You set the notional location of the nodes to `dc1` using `--location-names`. You then activate the PGD proxies in that location using `--active-locations dc1` set to the same location.
+You set the notional location of the nodes to `dc1` using `--location-names`. You then set `--pgd-proxy-routing` to `global` so that proxy routing can route traffic to all nodes in the cluster.
```

Comment:
Would it be more accurate to say something a little more specific? Global means writes are routed to a single write leader for the whole cluster, and local means they are routed to a local write leader.

Reply:
This is in the context of the quickstart, where we won't be discussing write leaders per se till deeper into the explore topics. It does make me consider modifying the explore content as a secondary update though.